### PR TITLE
Update Getting Started VRF tutorial from v1 to v2

### PR DESCRIPTION
**Pull request by aelmanaa**

### CODE REVIEW

File: docs/Introduction/tutorials/intermediates-tutorial.md [hunk 1]
```diff
@@ -344,62 +342,45 @@ contract VRFD20 is VRFConsumerBase, ConfirmedOwner(msg.sender) {
...
-- LINK address:     `0xa36085f69e2889c224210f603d836748e7dc0088`
-- keyHash:          `0x6c3699283bda56ad74f6b855546325b68d482e983852a7a82979cc4807b641f4`
-- fee:              `100000000000000000`
+Select the `VRFD20` contract or the name that you gave to your contract. You will deploy this contract on the Rinkeby test network.
 
-![Remix contract parameters to deploy](/files/intermediates-tutorial-03.png)
+Click the caret arrow on the right hand side of **Deploy** to expand the parameter fields, and paste your subscription id.
 
-Then click the `transact` button.
+![Remix contract parameters to deploy](/files/intermediates-tutorial-02.png)
```
Comment (dwightjl) [inline docs/Introduction/tutorials/intermediates-tutorial.md:359]: You might want to scale some of the larger images down a little bit if you have time.

File: docs/Introduction/tutorials/intermediates-tutorial.md [hunk 2]
```diff
@@ -344,62 +342,45 @@ contract VRFD20 is VRFConsumerBase, ConfirmedOwner(msg.sender) {
...
+## 7. How do I test `rollDice`?
 
-You will have to wait a few minutes for your transaction to confirm and the response to be sent back. You can get your house by clicking the `house` function button with your address. Once the response has been sent back, you'll be assigned a *Game of Thrones* house!
+After you open the deployed contract tab in the bottom left, the function buttons are available. Find `rollDice` and click the caret to expand the parameter fields. Enter an Ethereum address, and click 'rollDice'.
 
-You might notice that there are more functions listed than you originally built in the smart contract. These were inherited from the `VRFConsumerBase` and `ConfirmedOwner` smart contracts, which are used in the contract definition.
+You will have to wait a few minutes for your transaction to confirm and the response to be sent back. You can get your house by clicking the `house` function button with the address passed in `rollDice`. Once the response has been sent back, you'll be assigned a _Game of Thrones_ house!
```
Comment (dwightjl) [inline docs/Introduction/tutorials/intermediates-tutorial.md:381]:
```suggestion
You will have to wait a few minutes for your transaction to confirm and the response to be sent back. You can get your house by clicking the `house` function button with the address passed in `rollDice`. After the response is sent back, you'll be assigned a _Game of Thrones_ house!
```

File: docs/Introduction/tutorials/intermediates-tutorial.md [hunk 3]
```diff
@@ -344,62 +342,45 @@ contract VRFD20 is VRFConsumerBase, ConfirmedOwner(msg.sender) {
...
+Once you deploy your contract, it should be whitelisted in the subscription account so it can use the funding when requesting for randomness. Go to the [Subscription Manager](https://vrf.chain.link) and add your deployed contract address to the list of consumers. Find your contract address in Remix under **Deployed Contracts** on the bottom left.
 
-# 7. How do I test `rollDice`?
+![Remix contract address](/files/intermediates-tutorial-03.png)
 
-After you open the deployed contract tab in the bottom left, the function buttons are available. Find `rollDice` and click the caret to expand the parameter fields. Enter your Metamask address, and click 'roll'.
+## 7. How do I test `rollDice`?
 
-You will have to wait a few minutes for your transaction to confirm and the response to be sent back. You can get your house by clicking the `house` function button with your address. Once the response has been sent back, you'll be assigned a *Game of Thrones* house!
+After you open the deployed contract tab in the bottom left, the function buttons are available. Find `rollDice` and click the caret to expand the parameter fields. Enter an Ethereum address, and click 'rollDice'.
```
Comment (dwightjl) [inline docs/Introduction/tutorials/intermediates-tutorial.md:379]:
```suggestion
After you open the deployed contract tab in the bottom left, the function buttons are available. Find `rollDice` and click the caret to expand the parameter fields. Enter an Ethereum address to specify a "dice roller", and click 'rollDice'.
```

**Review by dwightjl — CHANGES_REQUESTED** [review]
Works great! Just a few edits.

File: _includes/samples/VRF/VRFD20.sol [hunk 4]
```diff
@@ -1,51 +1,66 @@
...
 
-import "@chainlink/contracts/src/v0.8/VRFConsumerBase.sol";
-import "@chainlink/contracts/src/v0.8/ConfirmedOwner.sol";
+import '@chainlink/contracts/src/v0.8/interfaces/VRFCoordinatorV2Interface.sol';
+import '@chainlink/contracts/src/v0.8/VRFConsumerBaseV2.sol';
 
 /**
  * @notice A Chainlink VRF consumer which uses randomness to mimic the rolling
- * of a 20 sided die
+ * of a 20 sided dice
```
[the remark this answers is not in the document]
Comment (aelmanaa) [reply _includes/samples/VRF/VRFD20.sol:9]: ok thanks , I learnt something :)

**Review by dwightjl — APPROVED** [review]

**Pull request merged**